.pre-commit-config.yaml

```diff
@@ -16,6 +16,7 @@ repos:
     hooks:
       - id: black
         args: [--target-version=py39]
+        exclude: docs/conf.py
```

Comment:
This file is 80% generated, we don't want to run linter on it.

Reply:
If this was a file that wasn't supposed to be manually modified but instead repeatedly re-generated, I'd agree with excluding it. In this case though, I suggest we run black on the generated output once and avoid an exception to this file just because it was generated initially.

I observed that the only differences were the use of double quotes compared to single quotes, so no manual interventions seems relevant to take along with running black.